src/main/resources/petriNets/dashboard_tile.xml

```diff
@@ -0,0 +1,1263 @@
+<document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://petriflow.com/petriflow.schema.xsd">
+    <id>dashboard_tile</id>
+    <initials>DST</initials>
+    <title name="model_title">Dashboard Tile</title>
```

Comment:
i18n keys are defined in the whole process but translations are missing

Reply:
done